substrate/frame/migrations/src/lib.rs

```diff
@@ -69,22 +69,22 @@
 //! either be [`MigrationCursor::Active`] or [`MigrationCursor::Stuck`]. In the active case it
 //! points to the currently active migration and stores its inner cursor. The inner cursor can then
 //! be used by the migration to store its inner state and advance. Each time when the migration
-//! returns `Some(cursor)`, it signals the pallet that it is not done yet.  
+//! returns `Some(cursor)`, it signals the pallet that it is not done yet.
```

Comment:
Its markdown, so this changes the formatting into one large blog:  

![Screenshot 2024-05-29 at 22 16 30](https://github.com/paritytech/polkadot-sdk/assets/10380170/8c7dc984-1ec8-4c4a-ac08-96b86f069519)